src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPage.java

```diff
@@ -65,11 +75,11 @@ private void tryToLoginUser(String clientAddress) {
 
         CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedbackPanel");
         loginForm.add(feedbackPanel);
-        RequiredTextField<String> loginField = new RequiredTextField<>(LOGIN_FIELD_ID);
+        RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
```

Comment:
By "one-time" I mean variables that are used only once.
To sum up, please refactor this (and similar in change scope - method in this case) into 
````java
loginForm.add(new RequiredTextField(LOGIN_FIELD_ID));
````
I see no point in existence of constructions like:
`````java
RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
loginForm.add(loginField);
````